integration/internal/build/build.go

```diff
@@ -43,10 +41,18 @@ func GetImageIDFromBody(t *testing.T, body io.Reader) string {
 		if jm.Aux == nil {
 			continue
 		}
-		assert.NilError(t, json.Unmarshal(*jm.Aux, &br))
-		assert.Assert(t, br.ID != "", "could not read image ID from build output")
-		break
+
+		var br build.Result
+		if err := json.Unmarshal(*jm.Aux, &br); err == nil {
+			if br.ID == "" {
+				continue
+			}
+			id = br.ID
+			break
+		}
 	}
 	io.Copy(io.Discard, body)
```

Comment:
If you still need to update; while we're updating; perhaps also explicitly discard the error;

```go
	_, _ = io.Copy(io.Discard, body)
```